.github/workflows/test.yml

```diff
@@ -0,0 +1,59 @@
+name: SROS2 CI
+on:
+  pull_request:
+  push:
+  schedule:
+    # Run daily
+    - cron:  '0 20 * * *'
+
+jobs:
+  test_latest:
+    runs-on: ubuntu-latest
+    container: osrf/ros2:devel
+    steps:
+    - run: |
+        apt-get -qq update
+        apt-get -qq upgrade -y
+        apt-get -qq install -y curl libasio-dev libtinyxml2-dev
+    - uses: mikaelarguedas/action-ros-ci@sros2-version
+      with:
+        package-name: |
+          sros2
+          sros2_cmake
+        # skipping end-to-end tests for now https://github.com/eProsima/Fast-RTPS/issues/1087
+        #  test_security
+        # extra-cmake-args: '-DSECURITY=ON --no-warn-unused-cli'
+    - name: Upload coverage to Codecov
+      uses: codecov/codecov-action@v1.0.6
+      if: github.event_name == 'pull_request' || (github.event_name == 'push' && github.ref == 'refs/heads/master')
+      with:
+        file: ros_ws/build/sros2/coverage.xml
+        flags: unittests
+        name: sros2-coverage
+        fail_ci_if_error: true
+        yml: .codecov.yml
+    - name: Upload Logs
+      uses: actions/upload-artifact@v1
+      if: failure()
+      with:
+        name: colcon-logs
+        path: ros_ws/log
+  test_nightly:
+    runs-on: ubuntu-latest
+    container: osrf/ros2:nightly
+    steps:
+    - uses: actions/checkout@v1
+    - run: |
+        apt-get -qq update
+        apt-get -qq upgrade -y
+        rosdep update
```

Comment:
No init needed?

Reply:
In short: all `ros:*` and `osrf/ros*` images install and init rosdep for you, so we should only ever `update`.


`init` is (or IMO should always be) done when the `rosdep` package is installed. As init cannot be ran twice without error, I believe users should not get in the habit of running it in scripts where they don't also handle the installation of rosdep.